src/components/home/balance-card.tsx

```diff
@@ -47,29 +56,30 @@ const BalanceCard = () => {
     setLoading(false);
   };
 
-  /**
-   * Fetch user's flextime data when users or userProfile changes.
-   */
-  useEffect(() => {
-    if (!usersFlextime) {
-      getUsersFlextimes();
-    }
-  }, [user]);
-
   /**
    * Render user's flextime data.
    */
   const renderUserFlextime = () => {
+    if (!usersFlextime?.totalFlextimeBalance) {
+      return <Skeleton />;
+    }
+    const totalFlextimeBalance = usersFlextime.totalFlextimeBalance;
+    const textColor = totalFlextimeBalance >= 0 ? "green" : "red";
+    const hourLabel =
+      totalFlextimeBalance === 1 ? strings.balanceCard.hour : strings.balanceCard.hours;
+
     return (
       <Typography variant="body1">
-        {strings.formatString(
-          strings.balanceCard.totalFlextimeBalance,
-          usersFlextime?.totalFlextimeBalance ?? ""
-        )}
+        {strings.balanceCard.totalFlextimeBalance}{" "}
+        <span style={{ color: textColor }}>{totalFlextimeBalance}</span> {hourLabel}
       </Typography>
     );
   };
 
+  if (!severaUserId) {
+    return null;
+  }
```

Comment:
I was hoping this could be controlled from the screens/home-screen like for the isDeveloper condition. 

You might need to check with Stephan to set up the severaID for your own user, for example granting yourself a severaId (probably the test one) and then seeing if it can be accessed through the access token in a similar way to the UserRoleUtils.isDeveloper